### ISSUE TITLE
Add multiple path source in esdoc config

### ISSUE THREAD

**Issue by buzzzzer** [esdoc/esdoc#86]
Is there a way to several paths, something like:
```json
{
   "source": [ "./path/to/firstsourcedir","./path/to/secondsourcedir"] 
}
```
?
Thx.

**Comment by h13i32maru** [comment]
@buzzzzer Hi! Please see the same [issue](https://github.com/esdoc/esdoc/issues/72#issuecomment-139860625)

**Comment by typhonrt** [comment]
I have created a comprehensive solution for multiple source path support with backward compatibility and an overview is available in #310.
